server/reviewsRoutes.js

```diff
@@ -3,7 +3,12 @@ const express = require('express');
 const reviewsRouter = express.Router();
 const reviewsController = require('./reviewsController');
 
-reviewsRouter.get('/:productId/', reviewsController.reviews);
-reviewsRouter.get('/:productId/:sortMethod', reviewsController.reviews);
+reviewsRouter.get('/product/:productId/', reviewsController.getReviews);
```

Comment:
I noticed that you use a product ID which is broken into a separate request. I was thinking maybe either the request for product data could be made at app Lvl, and state passed down to overview or I'll just get another route to get product Id only